packages/amazonq/src/app/inline/completion.ts

```diff
@@ -180,6 +182,12 @@ export class AmazonQInlineCompletionItemProvider implements InlineCompletionItem
         token: CancellationToken
     ): Promise<InlineCompletionItem[] | InlineCompletionList> {
         if (this.isNewSession) {
+            const isAutoTrigger = context.triggerKind === InlineCompletionTriggerKind.Automatic
+            if (isAutoTrigger && !CodeSuggestionsState.instance.isSuggestionsEnabled()) {
```

Comment:
Is this a bug on master as well?

Reply:
I see, wanted to double check this was hidden behind feature flag, but that makes sense.